framework-examples/angular4/README.md

```diff
@@ -0,0 +1,18 @@
+﻿# MDC-Web Example - Angular4
+
+This folder contains an example for using mdc-textfield and mdc-button within Angular4.
+
+## Setup
+
+1. Run `npm install` within this folder
+2. Run `npm start` to start the demo server
+3. Navigate to http://localhost:2000 to view the demo.
+
+## Notes
+
+The textfield wrapper is implemented within `src/app/components/textfield/textfield.ts`.
+The button wrapper is implemented within `src/app/components/button/button.ts`.
+
+## Todo
```

Comment:
I'd either remove this `Todo` and say that these buttons  don't have JS ripples, or just add it as part of this PR. I'd much rather not have a `Todo` as part of the README since it denotes that it's something the core team may be working on, which isn't the case :)